### PR TITLE
Next step towards the simulation challenge

### PR DESCRIPTION
**Pull request by victor-gonzalez**
Support for systems without centrality/multiplicity classes
Support for pp systems and the base for pPb, Pbp and XeXe
Support for MC reco analysis

### CODE REVIEW

File: Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx [hunk 1]
```diff
@@ -155,37 +177,114 @@ enum TrackPairs {
...
+    return kPbPb;
+  } else if (sysstr.compare("pp") == 0) {
+    return kpp;
+  } else if (sysstr.compare("pPb") == 0) {
+    return kpPb;
+  } else if (sysstr.compare("Pbp") == 0) {
+    return kPbp;
+  } else if (sysstr.compare("pPb") == 0) {
+    return kpPb;
+  } else if (sysstr.compare("XeXe") == 0) {
```
Comment (anerokhi) [inline Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx]: Why not just `sysstr == "abcd"`?

Comment (victor-gonzalez) [reply Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx]: You are right!
I guess I repeated what was needed when it is embedded within a `Configurable` but the cast in the function signature transforms it into a conventional `std::string`.
Thanks for pointing that out!
I modified the code accordingly

File: Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx [hunk 2]
```diff
@@ -237,6 +336,8 @@ struct DptDptCorrelationsFilterAnalysisTask {
   Configurable<int> cfgTrackType{"trktype", 1, "Type of selected tracks: 0 = no selection, 1 = global tracks FB96"};
   Configurable<bool> cfgProcessPairs{"processpairs", true, "Process pairs: false = no, just singles, true = yes, process pairs"};
   Configurable<std::string> cfgCentMultEstimator{"centmultestimator", "V0M", "Centrality/multiplicity estimator detector: default V0M"};
+  Configurable<std::string> cfgDataType{"datatype", "data", "Data type: data, MC, FastMC, OnTheFlyMC. Default data"};
+  Configurable<std::string> cfgSystem{"syst", "PbPb", "System: pp, PbPb, Pbp, pPb, XeXe. Default PbPb"};
 
   Configurable<o2::analysis::DptDptBinningCuts> cfgBinning{"binning",
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx:342]: Did you consider using AxisConfigurables?

**Review by jgrosseo — APPROVED** [review]

File: Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx (hunk 2 above)
Comment (victor-gonzalez) [reply Analysis/Tasks/PWGCF/dptdptcorrelations-simchl.cxx:342]: Not needed so far

**Pull request merged**